control_panel_api/services.py

```diff
@@ -19,6 +19,11 @@ def _policy_name(bucket_name, readwrite=False):
     return "{}-{}".format(bucket_name, READ_WRITE if readwrite else READ_ONLY)
 
 
+def _app_role_name(app_slug):
+    """Return the IAM role name for the given `app_slug`"""
+    return "{}_app_{}".format(settings.ENV, app_slug)
```

Comment:
In short no. The bucket name is something visible to the end user, the user enter the S3 bucket name directly and also needs to know the exact S3 bucket name (as it may write some code to read/write from it).

The IAM role here is something needed behind by the app to access the S3 bucket but its name is generated off the S3 bucket - not entered by the user. (but we still want to keep the same convention of prefixing it with the env name).

Let me know if this not clear and I can try to explain better 👍